synchronizer/synchronizer.go

```diff
@@ -528,8 +534,11 @@ func (s *Synchronizer) Sync(ctx context.Context,
 	log.Debugf("ethBlock: num: %v, parent: %v, hash: %v",
 		ethBlock.Num, ethBlock.ParentHash.String(), ethBlock.Hash.String())
 
-	if err := s.stats.UpdateEth(s.ethClient); err != nil {
-		return nil, nil, tracerr.Wrap(err)
+	if nextBlockNum+updateEthBlockNumThreshold >= s.stats.Eth.LastBlock.Num ||
```

Comment:
I suggest removing the time-based update approach in favor of this one. See https://github.com/hermeznetwork/hermez-node/pull/668#issuecomment-805780693

Reply:
Ok, I'm evaluating if this gives tangible performance gain after switching to fetching block headers.